web-client/integration-tests/externalUserFilesDocumentAcrossConsolidatedCases.test.ts

```diff
@@ -63,9 +63,39 @@ const verifyDocumentWasFiledAcrossConsolidatedCaseGroup = cerebralTest => {
   });
 };
 
+const verifyPractitionerAssociationAcrossConsolidatedCaseGroup = (
+  cerebralTest,
+  {
+    expectedAssociation,
+    practitionerRole,
+  }: { expectedAssociation: boolean; practitionerRole: string },
+) => {
+  return it(`should verify that ${practitionerRole} is ${
+    expectedAssociation ? '' : 'not'
+  } associated with all of the cases in the consolidated group`, () => {
+    const userId: string = cerebralTest.getState('user.userId');
+    const consolidatedCases = cerebralTest.getState(
+      'caseDetail.consolidatedCases',
+    );
+
+    consolidatedCases.forEach(aCase => {
+      const practitioners =
+        practitionerRole === 'irsPractitioner'
+          ? aCase.irsPractitioners
+          : aCase.privatePractitioners;
+      const isAssociated = !!practitioners.find(
+        practitioner => practitioner.userId === userId,
+      );
+
+      expect(isAssociated).toBe(expectedAssociation);
+    });
+  });
+};
+
 describe('External User files a document across a consolidated case group', () => {
   const cerebralTest = setupTest();
 
+  const leadCaseDocketNumber = '102-23';
```

Comment:
Does this mean this test can't be run twice without re-seeding the database?

Reply:
The test will start each run with seeding a set of seedData created specifically for this test. Because this seeding always occurs in the beforeAll,  we can ensure that each run has a pristine dataset. 

Also in the afterAll, we reseed our standard efcms-local seed data. So other tests will have the normal dataset they are looking for!

Reply:
Wonderful! Thanks!